src/request/dto/create-request.dto.ts

```diff
@@ -0,0 +1,117 @@
+import { RequestStatusEnum, RequestTypeEnum } from '@prisma/client';
```

Comment:
Need to import the ENUMs from a different folder.
path: "src/request/enum/request.enum.ts"

Reply:
tried that also but that was creating datatype issue for Prisma queries